### ISSUE TITLE
Collection Cards: Supplemental reports upload failing

### ISSUE THREAD

**Issue by ThorTheDestroyer** [TNRIS/data.tnris.org#183]
Can't seem to get the supplemental reports or the breaklines to upload to a new collection after selecting a zip.  Any ideas/suggestions?

**Comment by jwhaney** [comment]
i just created a test collection and was able to upload test zip files for both supplemental reports and breaklines.

**Comment by jwhaney** [comment]
make sure and save the new collection before uploading assets (.zip files or images).

**Comment by jwhaney** [comment]
another thing you may want to check is your zip file sizes. max file size for supplemental reports, breaklines, and tile indexes is 75mb.

just to be sure, i also tested this on a windows vm to make sure it's not something with windows OS messing with the zip file. seems to work just fine from what i can tell.

i'm gonna close this out now. feel free to swing by if you are still having trouble with any of it.

**Comment by ThorTheDestroyer** [comment]
@jwhaney The supplemental reports are larger than 75MB.  Is that a hard cap?  The zip file i have is about 220 MB.  If we need to exclude information that may be possible to make it smaller.

**Comment by jwhaney** [comment]
220mb is really big. when determining the 75mb cap, we looked at a lot of the collections with supp reports, breaklines, etc, and found that most max file sizes didn't exceed 20 - 30mb i believe. we gave extra room at 75mb but there really shouldn't be a reason to upload more than that.

i would suggest meeting with @gmullins to work on a set standard for what the supp report zipfiles should consist of. if we absolutely need to increase the max 75mb file size, then we can look at that possibly.

let me know if you have any questions.
